src/mindlessgen/molecules/generate_molecule.py

```diff
@@ -36,9 +37,9 @@ def generate_random_molecule(
     mol.xyz, mol.ati = generate_coordinates(
         at=mol.atlist,
         scaling=config_generate.init_coord_scaling,
-        dist_threshold=config_generate.dist_threshold,
```

Comment:
What happened to the "old" `config_generate.dist_threshold` entry in the CLI parser, `mindlessgen.toml`, the `ConfigClass`, ...?

Reply:
I let them untouched because i am not sure if the parameter `threshold` is needed in other routines. But if this parameter is not needed anymore i can delete it.

Reply:
Just find out if it's used somewhere else via `Find All References` in `code` (VS Code) or via `grep -rw "dist_threshold"` in the root folder of the repository source in the terminal.

Reply:
okay i should have removed the parameter `dist_threshold`